Documentation/getting-started.md

```diff
@@ -57,7 +57,23 @@ Kubernetes uses etcd for data storage and for cluster consensus between differen
 
 ### Single-Node/Development
 
-You can simply start etcd via [cloud-config][cloud-config-etcd] when you create your CoreOS machine or start it manually:
+You can simply start etcd via [cloud-config][cloud-config-etcd] when you create your CoreOS machine or start it manually.
+
+If you are starting etcd manually, we need to first configure it to listen on all interfaces:
+
+* Replace `${PUBLIC_IP}` with the etcd machines publically routable IP address.
+
+** /etc/systemd/system/etcd2.service.d/40-listen-address.conf
+
+```
+[Service]
+Environment=ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379
+Environment=ETCD_ADVERTISE_CLIENT_URLS=http://${PUBLIC_IP}:2379
+```
+
+Record the value used as the `ETCD_ADVERTISE_CLIENT_URLS`. In the rest of this guide, use that as the value for `ETCD_ENDPOINTS`.
```

Comment:
Use the value of `ETCD_ADVERTISE_CLIENT_URLS` as the value of `ETCD_ENDPOINTS` in the rest of this guide.